### DISCUSSION TITLE
【求教】编译错误

### DISCUSSION THREAD

**Discussion by XinghaiLuo** [Blinue/Magpie#133] [Q&A]
![image](https://user-images.githubusercontent.com/90503820/132943326-e641caca-44aa-4ab3-a4b5-328aac8fa942.png)
编译出错是因为什么啊。
求大佬教教我。

**Comment by Blinue** [comment]
请查看[编译指南](https://github.com/Blinue/Magpie/wiki/%E7%BC%96%E8%AF%91%E6%8C%87%E5%8D%97)

**Reply by XinghaiLuo** [reply]
您好，为什么我的exe文件点击之后没有任何反应
![image](https://user-images.githubusercontent.com/90503820/132985483-b8395185-bcf6-4a2f-a81e-df54d3df59e1.png)

**Reply by Blinue** [reply]
在VS里运行查看错误或者在[Release](https://github.com/Blinue/Magpie/releases)页面下载

**Reply by XinghaiLuo** [reply]
![image](https://user-images.githubusercontent.com/90503820/132987251-eb59dee8-caad-43fa-b094-75a4d3d7fb1b.png)
您好，在vs中是可以正常运行的
